File: 3275-minimum-number-of-pushes-to-type-word-i/minimum-number-of-pushes-to-type-word-i.cpp
class Solution {
public:
    int minimumPushes(string word) {
        int res = 0;

        for(int i = 0; i<word.length(); i++) {
            if(i < 8) res++;
            else if (i < 16) res += 2;
            else if (i < 24) res += 3;
            else res += 4;
        }

        return res;
    }
};

File: 3275-minimum-number-of-pushes-to-type-word-i/minimum-number-of-pushes-to-type-word-i.py
class Solution:
    def minimumPushes(self, word: str) -> int:
        res = 0
        n = len(word)
        i = 0

        while i < n :
            if i < 8 :
                res += 1
            elif i < 16 :
                res += 2
            elif i < 24 :
                res += 3
            else :
                res += 4
            
            i += 1
        
        return res
            